lib/rspec/core/configuration.rb

```diff
@@ -340,6 +340,11 @@ def exclude_pattern=(value)
       # @return [Symbol]
       add_setting :detail_color
 
+      # @macro add_setting
+      # Hyperlink failed examples (default: `false`).
+      # @return [Boolean]
+      add_setting :hyperlink
```

Comment:
I think this is a bad name, as its not clear what it does by name alone. If we merge this it should be something like `use_links_to_files_in_output` and should take either a custom scheme like `rspec` or default to `file`